spec/controllers/prison/dashboards_controller_spec.rb

```diff
@@ -15,7 +15,7 @@
 
     context "when logged in" do
       before do
-        sign_in user
+        allow(controller).to receive(:current_user).and_return(user)
```

Comment:
@alan could we redefine `sign_in` to do this, rather than repeating `allow(controller).to receive(:current_user).and_return(user)` in a zillion places?